src/components/Product/Product.js

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from 'react';
+import firebase from '@firebase/app';
+function Product() {
+  const [option, setOption] = useState('');
+  const [data, setData] = useState({
+    name: '',
+    lastDate: '',
+  });
+  const addProduct = (event) => {
+    event.preventDefault();
+    console.log('send data...' + data.name, data.lastDate, option);
+    firebase
+      .firestore()
+      .collection('products')
+      .add({
+        name: data.name,
+        lastDate: data.lastDate,
+        date: option,
+        token: token(),
```

Comment:
Let's remove this `token` from here. The `token` should be used as the collection name.

Reply:
Is corrected 💯